student.rb

```diff
@@ -0,0 +1,11 @@
+# student with there classroom and skills
+class Student < Person
```

Comment:
- [ ] For this inheritance to work you need to import the Person class into this file. Kindly do that.

  ```
  require_relative './person'
  ```